connector/spanmetricsconnector/README.md

```diff
@@ -33,6 +33,29 @@ across all spans:
 - `span.kind`
 - `status.code`
```

Comment:
Do we include `service.instance.id` by default? If yes, then it should be called out imo.